package/kedro_viz/launchers/cli.py

```diff
@@ -270,3 +273,49 @@ def deploy(region, bucket_name):
         )
     finally:
         viz_deploy_timer.terminate()
+
+
+@viz.command(context_settings={"help_option_names": ["-h", "--help"]})
+def build():
```

Comment:
I am not sure if we just want to copy the static build folder of kedro viz. Ideally we should create a build folder similar to what we copy to S3, something like [this](https://github.com/kedro-org/kedro-viz/blob/main/package/kedro_viz/integrations/deployment/s3_deployer.py#L112) and use it as a base folder for all the cloud providers

```python
self._upload_api_responses()
self._upload_static_files(_HTML_DIR)
self._upload_deploy_viz_metadata_file()
 ```